tests/unit/database/test_topic.py

```diff
@@ -384,6 +386,50 @@ def test_get_topics_for_pmc_article(test_data_path, monkeypatch):
     request_mock.assert_called_with("Journal NLM TA")
 
 
+@responses.activate
+def test_get_topics_for_arxiv_articles(test_data_path):
+    with open(test_data_path / "arxiv_api_response.xml") as f:
+        body = f.read()
+    id_queries = [
+        "id_list=q-bio%2F0401024v1%2Cq-bio%2F0401014v1%2C1808.02949v2",
+        "id_list=q-bio%2F0401024v1%2Cq-bio%2F0401014v1%2C1808.02949v2%2C1808.02950v7",
+    ]
+    for id_query in id_queries:
+        responses.add(
+            responses.GET,
+            f"http://export.arxiv.org/api/query?{id_query}&max_results=400",
```

Comment:
@jankrepl you looked into `responses` recently - wasn't there a new syntax for adding URL paramters?

Reply:
The problem we had was related to the function `responses.add_callback`. AFAIK the `responses.add` function should be fine.

https://github.com/BlueBrain/Search/pull/546

Reply:
I mean something like in the example here: https://github.com/getsentry/responses#query-parameters-as-a-dictionary Maybe there wasn't an explicit usecase for URL parameters in #546.

In the example I linked they do say "Note, do not use query parameters as part of the URL".

Reply:
oops, sorry, I misunderstood.

Yeh, one should use on of the options below instead of making it part of the URL
* `matchers.query_param_matcher`
* `matchers.query_string_matcher`

Reply:
Cool, I did not know about this new syntax!

Notice that we are already using `responses.add` with `URL` string arguments including query parameters.
So if we change it here let's do it everywhere.

Reply:
Also, notice that `responses` introduced these `matchers` in `0.14.0` (see [here](https://github.com/getsentry/responses/releases)) so we need to bump our requirements.

Reply:
Done in 72f1231.